django/contrib/admin/static/admin/img/README.txt

```diff
@@ -1,7 +1,18 @@
-All icons are taken from Font Awesome (https://fontawesome.com/) project.
-The Font Awesome font is licensed under the SIL OFL 1.1:
-- https://scripts.sil.org/OFL
+## Icons License
 
-SVG icons source: https://github.com/encharm/Font-Awesome-SVG-PNG
-Font-Awesome-SVG-PNG is licensed under the MIT license (see file license
-in current folder).
+All icons in this directory are provided by [Font Awesome Free](https://fontawesome.com), version 6.x.
```

Comment:
Why not providing a specific version?

Reply:
Because specifying version 6.x provides flexibility to choose any icon from the sixth major version. I could also include 5.x if we want to support icons from that version as well.

Reply:
+1 for being specific here. "6.x" creates the expectation of being always up to date with new 6.x releases, which I don't think will be the case.

Reply:
"All icons are taken from Font Awesome (https://fontawesome.com/) project."
"All icons in this directory are provided by [Font Awesome Free](https://fontawesome.com)"

On the other hand, do you think these changes are reasonable?

Reply:
The issue is that as soon as we omit a specific version, license compliance can become hard. Django essentially bundles a modified copy of a third-party library and linking to the unmodified library actually used here inside an own SBOM can be desired. This is much easier if Django provides the exact upstream version.

Reply:
However, this primarily concerns the icons, which are elements that are unlikely to change rapidly.